tests/build/jvm-build.go

```diff
@@ -247,103 +230,103 @@ var _ = framework.JVMBuildSuiteDescribe("JVM Build Service E2E tests", func() {
 				return true
 			}, timeout, interval).Should(BeTrue(), "timed out when waiting for the pipeline run to complete")
 		})
+		// Blocked by https://issues.redhat.com/browse/PLNSRVCE-519
```

Comment:
519 pruning should not have affected this ... now, PLNSRVCE-520 possiby could have

Reply:
in any event ... basically we can just readd all the tests again once all the infra-deployment / jvm-build-service changes are finally lined up